config/routes.rb

```diff
@@ -48,6 +49,14 @@
 
     resources :notes do
       resources :revisions, only: [:index, :show]
+
+      collection do
+        get :multi_destroy_status
```

Comment:
you can fuse this in one `collection` block.

Reply:
👍